views/default/contests.html

```diff
@@ -39,25 +39,26 @@
                  + '+05:30';
         }
 
-        /* Set Reminder in Google Calendar */
-        function setReminder(contestName, siteName, startTime, duration, contestLink) {
+        /* Authorize Google Calendar user */
+        function handleAuthorize(callback, showDialog) {
 
             // Your Client ID can be retrieved from your project in the Google
             // Developer Console, https://console.developers.google.com
             var CLIENT_ID = '{{=current.calendar_token}}';
             var SCOPES = ['https://www.googleapis.com/auth/calendar'];
 
-            var startTimeStamp = new Date(startTime.replace(/-/g , '/'));
-            var endTimeStamp = dateToTimeStamp(addMinutes(startTimeStamp, getMinutes(duration)));
-            startTimeStamp = dateToTimeStamp(startTimeStamp);
-
             /* Check if current user has authorized this application. */
             function checkAuth() {
-                gapi.auth.authorize({
+                if(gapi && gapi.auth) {
+                    gapi.auth.authorize({
                         'client_id': CLIENT_ID,
-                        'scope': SCOPES.join(' '),
```

Comment:
This was used to make scope argument a string. This is required. Though it works without this I would prefer the former one as used [here](https://developers.google.com/google-apps/calendar/quickstart/js)